_lessons/16-setting-up-the-layout.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Setting up the Layout
+lesson: 16
+layout: default
+---
+
+Welcome to your first project! From this point on, there will be no more "Try it Yourself" boxes in the tutorial. Instead, you should follow along with everything on your own computer. You can also download the files for each project step on Github. 
+
+The lessons for this project will be a bit longer than in the previous sections, so be sure to pace yourself and take all the time you need to follow along. 
+
+### Set up the Project File
+
+The first thing we'll do, before we start writing any project code, is to create an HTML file with places to put our HTML and CSS. To recap, here is the standard structure of an HTML file:
+
+```
+<!DOCTYPE html>
+<html>
+  <head>
+    <style type="text/css">
+      (Your CSS goes here!)
+    </style>
+  </head>
+  <body>
+      (Your HTML content goes here!)
+  </body>
+</html>
+```
+
+The CSS for our project will go in-between the `style` tags, and the HTML for everything displayed on the page will go in between the `body` tags. 
+
+### Code the Layout with Divs
+
+Before we start writing any code, we should first draw what we want our basic layout to look like. For this project, we will have a header, footer, sidebar and content. We'll place the header and footer at the top and bottom, and we'll place the sidebar and content side-by-side in between. The sidebar will go on the left, and content on the right. 
+
+(img)
+
+The way to translate this into code is to think of each section as its own `div`. To make it clear what each `div` is, we can give it a custom ID. So we can start our code by writing a `div` for each section: 
+
+```
+<div id="header">Header</div>
+<div id="sidebar">Sidebar</div>
+<div id="content">Content</div>
+<div id="footer">Footer</div>
+```
+
+Next, we think about what we need to do to arrange the divs. By default, divs are stacked one after another, vertically. However, we want the sidebar and content to be side-by-side. Remember that in the last section, we saw a way to place two divs side-by-side by putting them both inside a third div, and then setting some CSS (`display: flex;`) on the div they are inside of. 
+
+Let's do this now, and give this div a descriptive ID as well - how about "container"?
+
+```
+<div id="header">Header</div>
+<div id="container">
+  <div id="sidebar">Sidebar</div>
+  <div id="content">Content</div>
+</div>
+<div id="footer">Footer</div>
+```
+
+And we'll add a CSS rule adding the flexbox rule to the container: 
+
+```
+#container {
+  display: flex; 
+}
+```
+
+Now if we load the page, we see it looks a little more like what we were expecting. 
+
+<div class="screenshot">
+    <img src="/assets/ch16_2.png" />
+</div>
```

Comment:
This image is a bit chronologically inconsistent; we haven't actually added a border to anything yet.